pkg/services/publicdashboards/service/testData/dashboardWithAnnotations.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
do we need a test data directory? Also, I don't see this file used anywhere.

Reply:
yeah it was used. Not used anymore though. Will delete.